### PR TITLE
C51-45: Case List Sorting Markup

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this the Markup has been updated for the Case List's Sorting to follow the new [design](https://projects.invisionapp.com/d/#/console/13827931/288222702/preview).

## Before
2018-09-04 at 9 03 am

## After
after

## Technical Discussion
1. Updated HTML in `CaseListTable.html` for the sorting ui.
2. Update sorting related classes to follow BEM.
3. Commented the css code in `css/caseList.css`. This is to make sure the css does not affect the UI unintentionally.
The following classes has been added
**civicase__case-list-column--first**
**civicase__case-list-sortable-header** - Sortable table headers
**civicase__case-list__toggle-sort** - Button beside the dropdown to toggle sorting direction
**civicase__case-list__header-toggle-sort** - Icons inside table headers to toggle sorting direction 
**civicase__case-sort-dropdown** - The dropdown to select the sorting sorting column.

### CODE REVIEW

File: scss/components/_case-list-table.scss [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
     th:first-child {
       background: $gray-lighter;
-      height: 61px;
+      height: 63px;
```
Comment (swastikpareek) [inline scss/components/_case-list-table.scss]: Any reason for this magic number. Can't we accommodate the height using padding and line-heights?

**Comment by swastikpareek** [comment]
@deb1990 : Can we have the same loading state on the browser as we have at the time of loading the content for the first time, when we change the sorting criterion?

File: scss/components/_case-list-table.scss (hunk 1 above)
Comment (deb1990) [reply scss/components/_case-list-table.scss]: I just updated the old one, as the UI was broken. Looks like, if I dont mention the height here, it looks fine. will do that. thanks

**Comment by deb1990** [comment]
> @deb1990 : Can we have the same loading state on the browser as we have at the time of loading the content for the first time, when we change the sorting criterion?

@swastikpareek  this PR is about the HTML only. Functionality has not been touched here. But I agree that its a good point. I will try to fix this as part of C51-46.

**Comment by deb1990** [comment]
@swastikpareek PR rebased, please approve again

**Comment by swastikpareek** [comment]
PR looks goods to me. Approved!

**Pull request merged**